pint/default_en.txt

```diff
@@ -538,23 +538,31 @@ neper = 1 ; logbase: 2.71828182845904523536028747135266249775724709369995; logfa
     cubic_yard = yd ** 3 = cu_yd
 @end
 
+# Note: historically, many of the units in this group were derived from each other (e.g. 1 chain = 10 fathoms = 4 rods = 100 links)
+# but we've followed the NIST's lead here and defined all of the units directly based on "common" units (feet and miles)
+# Source: Warfield, L. , Konijnenburg, J. , Lippa, K. , Lee, G. , Minnich, L. and Williams, J. (2023), Specifications, Tolerances, and Other Technical Requirements for Weighing and Measuring Devices as adopted by the 108th National Conference on Weights and Measures, Handbook (NIST HB), National Institute of Standards and Technology, Gaithersburg, MD, [online], https://doi.org/10.6028/NIST.HB.44-2024 (Accessed January 12, 2024)
 @group USCSLengthSurvey
-    link = 1e-2 * chain = li = survey_link
-    survey_foot = 1200 / 3937 * meter = sft
-    fathom = 6 * survey_foot
-    rod = 16.5 * survey_foot = rd = pole = perch
-    chain = 4 * rod
-    furlong = 40 * rod = fur
-    cables_length = 120 * fathom
-    survey_mile = 5280 * survey_foot = smi = us_statute_mile
-    league = 3 * survey_mile
-
-    square_rod = rod ** 2 = sq_rod = sq_pole = sq_perch
-    acre = 10 * chain ** 2
-    square_survey_mile = survey_mile ** 2 = _ = section
+    cables_length = 720 * foot
+    chain = 66 * foot = ch
+    fathom = 6 * foot
+    furlong = 660 * foot = fur
+    league = 3 * mile
+    link = 0.66 * foot = li = survey_link
+    rod = 16.5 * foot = rd = pole = perch
+
```

Comment:
Please keep the order from small to big length unit to make diffs simpler to follow